GraphView/po/pl-local.po

```diff
@@ -0,0 +1,634 @@
+msgid ""
+"Graphview Warning:  GraphViz (http://www.graphviz.org) is required for this "
+"view to work"
+msgstr ""
+"Ostrzeżenie GraphView: Do działania wymagany jest GraphViz"
+"(http://www.graphviz.org)"
+
+#: GraphView/graphview.gpr.py:63
+msgid "Graphview Failed to Load"
+msgstr "GraphView nie został załadowany"
+
+#: GraphView/graphview.gpr.py:64
+#, python-format
+msgid ""
+"\n"
+"\n"
+"Graphview is missing python modules or programs.\n"
+"%smust be installed.\n"
+"\n"
+"For now, it may be possible to install the files manually. See\n"
+"<a href=\"https://gramps-project.org/wiki/index.php?title=Graph_View\" title="
+"\"https://gramps-project.org/wiki/index.php?title=Graph_View\">https://"
+"gramps-project.org/wiki/index.php?title=Graph_View</a> \n"
+"\n"
+"To dismiss all future Graphview warnings click Dismiss."
+msgstr ""
+"\n"
+"\n"
+"W GraphView brakuje modułów lub programów Pythona.\n"
+"%s musi być zainstalowane.\n"
+"\n"
+"Na razie możliwe jest ręczne zainstalowanie plików. Zobacz\n"
+"<a href=\"https://gramps-project.org/wiki/index.php?title=Graph_View\" title="
+"\"https://gramps-project.org/wiki/index.php?title=Graph_View\">https://"
+"gramps-project.org/wiki/index.php?title=Graph_View</a> \n"
+"\n"
+"Aby odrzucić wszystkie przyszłe ostrzeżenia GraphView, kliknij Odrzuć."
+
+#: GraphView/graphview.gpr.py:80
+msgid " Dismiss "
+msgstr " Odrzuć "
+
+#: GraphView/graphview.gpr.py:81
+msgid "Continue"
+msgstr "Kontynuuj"
+
+#: GraphView/graphview.py:681 GraphView/graphview.py:3567
+msgid "Show images"
+msgstr "Pokaż obrazki"
+
+#: GraphView/graphview.py:684
+msgid "Show avatars"
+msgstr "Pokaż awatary"
+
+#: GraphView/graphview.py:687 GraphView/graphview.py:3575
+msgid "Highlight the home person"
+msgstr "Wyróżnij osobę główną"
+
+#: GraphView/graphview.py:691 GraphView/graphview.py:3583
+msgid "Show full dates"
+msgstr "Pokaż pełne daty"
+
+#: GraphView/graphview.py:695 GraphView/graphview.py:3591
+msgid "Show places"
+msgstr "Pokaż miejsca"
+
+#: GraphView/graphview.py:698 GraphView/graphview.py:2877
+msgid "Default"
+msgstr "Domyślny"
+
+#: GraphView/graphview.py:704
+msgid "Place format"
+msgstr "Format miejsc"
+
+#: GraphView/graphview.py:709 GraphView/graphview.py:3599
+msgid "Show tags"
+msgstr "Pokaż tagi"
+
+#: GraphView/graphview.py:711
+msgid "Layout"
+msgstr "Layout"
+
+#: GraphView/graphview.py:729
+msgid "Person theme"
+msgstr "Motyw osoby"
+
+#: GraphView/graphview.py:734
+msgid "Path color to home person"
+msgstr "Kolor ścieżki do osoby domowej"
+
+#: GraphView/graphview.py:738
+msgid "Font:"
+msgstr "Czcionka:"
+
+#: GraphView/graphview.py:752
+msgid "Avatars style"
+msgstr "Styl awatarów"
+
+#: GraphView/graphview.py:758
+msgid "PNG files"
+msgstr "Plik PNG"
+
+#: GraphView/graphview.py:763
+msgid "Male avatar:"
+msgstr "Awatar męski:"
+
+#: GraphView/graphview.py:764
+msgid "Choose male avatar"
+msgstr "Wybierz męski awatar"
+
+#: GraphView/graphview.py:776
+msgid "Female avatar:"
+msgstr "Awatar damski:"
```

Comment:
We should check how the female is translated in Gramps. If I recall correctly it was "kobieta/kobiecy" rather than "damski".

Reply:
You're right. Correct.